astropy/modeling/functional_models.py

```diff
@@ -506,7 +506,7 @@ def _parameter_units_for_data_units(self, inputs_unit, outputs_unit):
                             ('amplitude', outputs_unit['z'])])
 
 
-class Shift(Model):
+class Shift(Fittable1DModel):
```

Comment:
I should probably have added a `fit_deriv` for Scale while I was at it, otherwise we may get questions about why one works but not the other (there are probably other examples but those two go together).